api/members/router.ts

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+import KoaJoiRouter from 'koa-joi-router';
+import Service from './service'
+
+const router = KoaJoiRouter();
+// TODO: this
+// const Joi = KoaJoiRouter.Joi;
+
+router.prefix('/members');
+
+router.route({
+    path: '/',
+    method: 'GET',
+    handler: Service.listMembers
+});
```

Comment:
The Swagger merge added some additional work that needs to be done here. Please check updates to CONTRIBUTING.md and the other routers as a reference. You'll also need to create a `definitions.ts` file.